daisy_workflows/image_import/inspection/inspect_disk.py

```diff
@@ -14,6 +14,22 @@
 # limitations under the License.
 
 # Example of communicating a key-value pair back to Daisy:
-#   print("Status: <serial-output key:'partition_scheme' value:'mbr'>")
+#
+
+import os
+
+try:
+  os.system('fdisk -l /dev/sdb -o type')
+  stream = os.popen('fdisk -l /dev/sdb -o type')
+  output = stream.read()
+
+  # 1. For GPT, the ESP output should be "EFI System", which matches partition
+  # GUID "C12A7328-F81F-11D2-BA4B-00A0C93EC93B".
```

Comment:
Why is this ID in the code?

Reply:
So is this an example of a GUID or something that's always the same? If it's an example, make sure to document it as such.